src/gc/gc.cpp

```diff
@@ -30951,8 +30947,8 @@ void gc_heap::set_mem_verify (uint8_t* start, uint8_t* end, uint8_t b)
 #ifdef VERIFY_HEAP
     if (end > start)
     {
-        if ((g_pConfig->GetHeapVerifyLevel() & EEConfig::HEAPVERIFY_GC) &&
-           !(g_pConfig->GetHeapVerifyLevel() & EEConfig::HEAPVERIFY_NO_MEM_FILL))
+        if ((GCConfig::GetHeapVerifyLevel() & EEConfig::HEAPVERIFY_GC) &&
```

Comment:
You may need to cache the frequently accessed values like the GCHeapVerifyLevel. Fetching them by name each time is going to be too slow.